Project.toml

```
@@ -27,11 +27,11 @@ CFTime = "0.1"
 ClimaComms = "0.3, 0.4, 0.5"
 ClimaCore = "0.10.44"
 ClimaCoreTempestRemap = "0.3.5"
-DiffEqCallbacks = "2"
 DocStringExtensions = "0.8, 0.9"
 IntervalSets = "0.5, 0.6, 0.7"
 JLD2 = "0.4"
 NCDatasets = "0.11, 0.12"
+SciMLBase = "<1.51"
```

```suggestion
SciMLBase = "1"
```
Is the <1.51 for a known incompatibility before that version? If so, then feel free to skip the suggestion! I imagine nobody will resolve to belle that version anyway since it’s relatively old

---

We had this compat in the docs/ and experiments/ Projects before, but I'm not sure why. I'll change it to 1